src/main/java/com/onarandombox/MultiverseNetherPortals/listeners/MVNPEntityListener.java

```diff
@@ -166,7 +169,12 @@ private Location getLocation(Entity e, Location currentLocation, PortalType type
                     }
                 }
 
-                newTo = this.linkChecker.findNewTeleportLocation(currentLocation, destinationWorld, e);
+                if (shouldAppearAtSpawn) {
+                    MultiverseWorld tpTo = this.worldManager.getMVWorld(destinationWorld);
+                    newTo = this.linkChecker.findNewTeleportLocation(tpTo.getSpawnLocation(), destinationWorld, e);
```

Comment:
I see, i unfortunately dont know how to do that, this can only be a problem when traveling living entiities, cause items will just go up the blocks or to some random side which is kinda expected but living creatures will die of suffocation. I also wasn't able to identify somewhere else in the codebase where a safe teleporter is used, can i get some help?

Reply:
Hello?

Reply:
https://github.com/Multiverse/Multiverse-Core/blob/2d059eabc39c00e786e8dadf1e53c6f66d5a05a9/src/main/java/com/onarandombox/MultiverseCore/utils/WorldManager.java#L599 is an example of using the safe teleporter

Reply:
I made it use safe teleport when going to spawnpoint only. hope this is enough

Reply:
I’ll re-request a review from dump